docs/guides/how_to_guides/creating_batches/how_to_load_a_spark_dataframe_as_a_batch.rst

```diff
@@ -159,7 +157,7 @@ This guide will help you load a Spark DataFrame as a Batch for use in creating E
                     datasource_name="insert_your_spark_datasource_name_here",
                     data_connector_name="insert_your_runtime_data_connector_name_here",
                     batch_data=insert_your_dataframe_here,
-                    data_asset_name="IN_MEMORY_DATA_ASSET",
+                    data_asset_name="optionally_insert_your_data_asset_name_here",
```

Comment:
Same concern as above.

Reply:
I'll close this since it's been changed in the docs to be a required parameter :)